crates/cli/commands/src/db/mod.rs

```diff
@@ -52,6 +53,8 @@ pub enum Subcommands {
     Version,
     /// Returns the full database path
     Path,
+    /// Runs a bootnode for the discovery protocol
+    Bootnode(bootnode::Command),
```

Comment:
this doesn't quite fit under db commands, can make this a standalone command instead and then enable for the reth/op cli struct instead:

https://github.com/paradigmxyz/reth/blob/main/bin/reth/src/cli/mod.rs#L211-L211